tools/packaging/kernel/patches/5.15.x/0001-s390-ap-enable-sysfs-attribute-scans-to-force-AP-bus.patch

```diff
@@ -0,0 +1,44 @@
+From 837cd1059a985ee610e4b72dcdddb287e7b97de0 Mon Sep 17 00:00:00 2001
+From: Harald Freudenberger <freude@linux.ibm.com>
+Date: Wed, 16 Feb 2022 14:26:48 +0100
+Subject: [PATCH] s390/ap: enable sysfs attribute scans to force AP bus rescan
```

Comment:
> Hm... ok. I don't love merging this in Kata before the change is merged in a released kernel upstream. Red Hat, for one, deploys Kata with a packaged guest kernel rather than something custom built with these patches.

This patch has been merged into 5.17-rc3 (https://git.kernel.org/pub/scm/linux/kernel/git/torvalds/linux.git/commit/drivers/s390/crypto/ap_bus.c?id=8944d05f9bbf910c8b241e29a3de114900e31e42)

Reply:
Thanks @hbrueckner -- the included patch is now as released on kernel.org.